sql/core/src/test/resources/sql-tests/inputs/ansi/literals.sql

```diff
@@ -0,0 +1,2 @@
+--- malformed interval literal with ansi mode
```

Comment:
hmm, do we still have interval related tests in `literals.sql`?

Reply:
Is it ok to move all the interval-related queries here into `interval.sql` or `ansi/interval.sql`?

Reply:
I think so. @yaooqinn do you have time to take it later? IIRC there are a few interval tests in `group-by.sql` as well.

Reply:
ok. As for `literals.sql`, I'll move all the interval-related queries into correct files in this pr.

Reply:
some cases are using `interval ` but to verify `literals` such as
```
-- awareness of the negative sign before type
select -integer '7';
select -date '1999-01-01';
select -timestamp '1999-01-01';
select -x'2379ACFe';
select +integer '7';
select +interval '1 second';
```

the `group-by.sql `contains  avg and sum support, which should be move to `interval.sql`